package.json

```diff
@@ -13,6 +13,7 @@
     "firebase": "^2.2.7",
     "lodash": "^3.10.1",
     "react-google-charts": "^0.1.8",
+    "react-dom": "^0.14.3",
```

Comment:
`"react-dom": "^0.14.3",` gotta come out... not sure how that got back in there :)